### PR TITLE
Remove sensio generator

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is a BC break.

## Changelog

```markdown
### Removed
- Remove sensio generator
- Remove `CreateClassCacheCommand`
```

### CODE REVIEW

**Comment by core23** [comment]
After removing https://packagist.org/packages/sensio/generator-bundle, we are finally testing symfony 4+. The *dev*-dependency had locked symfony to version 3 oO

**Comment by greg0ire** [comment]
> After removing https://packagist.org/packages/sensio/generator-bundle, we are finally testing symfony 4+. The dev-dependency had locked symfony to version 3 oO

Ouch! Is that why Travis is failing? We should probably add some sanity check for this :sweat_smile:

**Comment by core23** [comment]
> > After removing https://packagist.org/packages/sensio/generator-bundle, we are finally testing symfony 4+. The dev-dependency had locked symfony to version 3 oO
> 
> Ouch! Is that why Travis is failing? We should probably add some sanity check for this 😅

No idea. Btw. we have no travis tests for symfony 4 🤔

> Looks good, but I think it's far from being the end of it after looking at the tests 😅

I wonder why this code was working, because in production you could use symfony 4

**Comment by greg0ire** [comment]
> No idea. Btw. we have no travis tests for symfony 4 thinking

That's probably something we usually add when a new major gets out, and we should probably challenge that.

**Comment by core23** [comment]
We need to port some removed test classes here too

File: src/Controller/CRUDController.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ public function setContainer(ContainerInterface $container = null): void
      *
      * @deprecated since version 3.27, to be removed in 4.0. Use Sonata\AdminBundle\Controller\CRUDController::renderWithExtraParams() instead.
      */
-    public function render($view, array $parameters = [], Response $response = null)
+    public function render(string $view, array $parameters = [], ?Response $response = null): Response
```
Comment (jordisala1991) [inline src/Controller/CRUDController.php]: This method should be removed anyway

Comment (core23) [reply src/Controller/CRUDController.php]: That's right, but I wanted to make the removal more atomic to have a shorter review process

**Comment by core23** [comment]
Can someone please help? Why is the method `AbstractWidgetTestCase::getEnvironment` final? Can't find the exact version where this method is/was final

**Comment by franmomu** [comment]
> Can someone please help? Why is the method `AbstractWidgetTestCase::getEnvironment` final? Can't find the exact version where this method is/was final

Could it be when [importing from CoreBundle](https://github.com/sonata-project/form-extensions/commit/793b43b3d7b07161c3e3ffa9d15e4e6bae55bb8e#diff-c6beb07a86f5ddd0544c34b0082df27b)?

**Comment by core23** [comment]
Temporary removed the CoreBundle test from Travis, because this will be removed with the next PR https://github.com/sonata-project/SonataAdminBundle/pull/5787

**Pull request merged**